SpotifySearch/uiImageExtension.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  uiImageExtension.swift
```

Comment:
Properly name your extension with capital camel-case. For example `UIImageView + DownloadImage`